Voice Note/Views/BottomBarView.swift

```diff
@@ -119,7 +123,35 @@ struct BottomBarView: View {
             )
         }.ignoresSafeArea(.all)
     }
-    
+    func saveVoiceNote() {
+        guard let url = voiceNoteViewModel.fileUrlList.last else {
+            return
+        }
+
+        let newVoiceNote = VoiceNote(context: managedObjectContext)
+            newVoiceNote.id = UUID()
+            newVoiceNote.text = speechRecognizer.transcriptionText
+            newVoiceNote.title = "Note Title"
+            newVoiceNote.fileUrl = url
+            newVoiceNote.createdAt = Date()
+            newVoiceNote.duration = 3765
```

Comment:
Can't we use getDuration(for: url) function of VoiceNoteViewModel? it is private, do not know if it breaks app functionality if we make it internal/public and use it here in BottomBarView. This function is not used any where in the project.

Reply:
Sure, we should implement that function in this case.